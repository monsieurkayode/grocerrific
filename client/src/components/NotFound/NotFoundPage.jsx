import React from 'react';
import { Link } from 'react-router-dom';

// eslint-disable-next-line import/no-named-as-default
import Header from '../common/Header';
import NoContent from '../common/NoContent';

const NotFoundPage = () => (
  <main id="inventory" className="container">
    <Header>
      <div className="header__right">
        <Link to="/store">
          <div className="start text__medium_bold uppercase">
            Back to store
          </div>
        </Link>
      </div>
    </Header>
    <section id="inventory__display">
      <NoContent
        content="404! You have reached the end of the rainbow :-)"
      />
    </section>
  </main>
);

export default NotFoundPage;
