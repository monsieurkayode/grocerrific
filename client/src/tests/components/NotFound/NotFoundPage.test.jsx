import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../../components/NotFound/NotFoundPage';

describe('NotFoundPage Component', () => {
  const wrapper = shallow(<NotFoundPage />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
