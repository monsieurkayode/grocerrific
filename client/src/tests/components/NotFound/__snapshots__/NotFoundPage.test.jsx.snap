// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Component renders correctly 1`] = `
<main
  className="container"
  id="inventory"
>
  <withRouter(Header)>
    <div
      className="header__right"
    >
      <Link
        replace={false}
        to="/store"
      >
        <div
          className="start text__medium_bold uppercase"
        >
          Back to store
        </div>
      </Link>
    </div>
  </withRouter(Header)>
  <section
    id="inventory__display"
  >
    <NoContent
      content="404! You have reached the end of the rainbow :-)"
    />
  </section>
</main>
`;
